contract_manager/src/sui.ts

```diff
@@ -209,6 +209,33 @@ export class SuiContract extends Contract {
     return this.executeTransaction(tx, keypair);
   }
 
+  async executeCreatePriceFeeds(keypair: Ed25519Keypair, vaa: Buffer) {
```

Comment:
we should also add a similar function `executeCreatePriceFeedsUsingAccumulator`, which calls [create_price_feeds_using_accumulator](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/sui/contracts/sources/pyth.move#L100C16-L100C52), but this can be done in a later PR.